src/services/execution-api/service.ts

```diff
@@ -173,11 +389,57 @@ export const makeExecutionApi = (
     }
   }
 
+  const resolveConsensusAddress = async () => {
```

Comment:
do we want to check consensus contract address against the address in the `ConsensusReached` event in case HashConsensus is changed during the Ejector job period